core/vm/lfvm/instructions.go

```diff
@@ -104,12 +110,20 @@ func opPush(c *context, n int) {
 }
 
 func opPush1(c *context) {
+	if c.stack.stack_ptr >= len(c.stack.data) {
```

Comment:
nit: could be a member function of the stack: `if c.stack.full() {`

Reply:
done